sdk/maps/azure-maps-search/pom.xml

```diff
@@ -109,5 +114,11 @@
       <version>3.4.30</version> <!-- {x-version-update;io.projectreactor:reactor-test;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core</artifactId>
+      <version>1.42.0-beta.1</version>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
remove?

Reply:
Yeah, I must have added that with IntelliJ by accident, thanks for the catch